test/libyul/yulOptimizerTests/loopInvariantCodeMotion/no_move_memory_msize.yul

```diff
@@ -0,0 +1,35 @@
+{
+  let b := 1
+  let c := msize() // prevents moving keccak and mload
```

Comment:
Why is that?

Reply:
because reading memory modifies msize

Reply:
But `keccak` could be moved after this

Reply:
"sane" code does not use `msize`, so it's fine to do nothing.